docs/deployment/connect-scm.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: connect-scm 
+title: Connect a source code manager
+hide_title: true
+description: Connect a GitHub or GitLab organization to manage user authentication.
+tags:
+  - Semgrep Cloud Platform
+  - Team & Enterprise Tier
+  - Deployment
+---
+
+# Connect a source code manager
+
+:::note Your deployment journey
+- You have [created a Semgrep account and organization](/deployment/create-account-and-orgs). 
+:::
+
+Linking a source code manager provides the following benefits:
+
+- Allows the Semgrep org membership to be managed by GitHub or GitLab.
+- For GitHub users:
+    - Provides Semgrep access to post PR or MR comments.
+    - For GitHub Actions users: Enables you to add a Semgrep CI job to repositories in bulk.
```

Comment:
```suggestion
    - Enables you to add a Semgrep CI job to repositories in bulk using GitHub Actions.
```

Reply:
I'd rather keep GHA first in the sentence because this feature is exclusively available to those users. Resolving as Won't Fix.